Documentation/user-guides/getting-started.md

```diff
@@ -0,0 +1,156 @@
+
+The `Prometheus` resource includes a selection of `ServiceMonitors` to be used, this field is called the `serviceMonitorSelector`.
+
+First, deploy three instances of a simple example application, which listens and exposes metrics on port `8080`.
+
+[embedmd]:# (../../example/user-guides/getting-started/example-app-deployment.yaml)
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: example-app
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: example-app
+    spec:
+      containers:
+      - name: example-app 
+        image: fabxc/instrumented_app
+        ports:
+        - name: web
+          containerPort: 8080
+```
+
+The `ServiceMonitor` has a label selector to select `Service`s and the underlying `Endpoints` objects. The `Service` object for the example application selects the `Pod`s by the `app` label having the `example-app` value. In addition to that the `Service` object specifies the port the metrics are exposed on.
+
+[embedmd]:# (../../example/user-guides/getting-started/example-app-service.yaml)
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: example-app
+  labels:
+    app: example-app
+spec:
+  selector:
+    app: example-app
+  ports:
+  - name: web
+    port: 8080
+```
+
+This `Service` object is discovered by a `ServiceMonitor`, which selects in the same way. The `app` label must have the value `example-app`.
+
+[embedmd]:# (../../example/user-guides/getting-started/example-app-service-monitor.yaml)
+```yaml
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: ServiceMonitor
+metadata:
+  name: example-app
+  labels:
+    team: frontend
+spec:
+  selector:
+    matchLabels:
+      app: example-app
+  endpoints:
+  - port: web
+```
+
+Finally, a `Prometheus` object defines the `serviceMonitorSelector` to specify which `ServiceMonitor`s should be included. Above the label `team: frontend` was specified, so that's what the `Prometheus` object selects by.
+
+[embedmd]:# (../../example/user-guides/getting-started/prometheus-example.yaml)
+```yaml
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: Prometheus
+metadata:
+  name: example
+spec:
+  serviceMonitorSelector:
+    matchLabels:
+      team: frontend
+  resources:
+    requests:
+      memory: 400Mi
+```
+
+This way the frontend team can create new `ServiceMonitor`s and `Service`s resulting in `Prometheus` to be dynamically reconfigured.
+
+To be able to access the Prometheus instance it will have to be exposed to the outside somehow. For demonstration purpose it will be exposed via a `Service` of type `NodePort`.
+
+[embedmd]:# (../../example/user-guides/getting-started/prometheus-example-service.yaml)
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus-example
+spec:
+  type: NodePort
+  ports:
+  - name: web
+    nodePort: 30900
+    port: 9090
+    protocol: TCP
+    targetPort: web
+  selector:
+    prometheus: example
+```
+
```

Comment:
Can you add the kubectl command to start this? (mostly for consistency with above)